ParkJInjoo/package.json

```
@@ -0,0 +1,23 @@
+{
+  "name": "parkjinjoo",
+  "version": "1.0.0",
+  "description": "",
+  "main": "app.js",
+  "scripts": {
+    "start": "nodemon app.js",
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "cors": "^2.8.5",
+    "dbmate": "^1.0.3",
+    "dotenv": "^16.0.3",
+    "express": "^4.18.2",
+    "morgan": "^1.10.0",
+    "mysql2": "^2.3.3",
+    "nodemon": "^2.0.20",
+    "typeorm": "^0.3.10"
+  }
```

`dependencies`와 `devDependencies`의 차이점을 명확히 구분해주는 것이 좋습니다!

실제 배포 단계에서 필요한 패지키 모듈은 `dependencies`에,
개발 환경에서만 필요한 패키지 모듈은 `devDependencies`에 추가해주세요!

추가로 dbmate는 전역으로 설치한 클라이언트를 사용하고 있을 것입니다!
따라서 npm에서 제공하는 dbmate 패키지는 설치할 필요가 없어보입니다!

- npm install "package-name"
- npm install --save-dev "package-name"
- npmu uninstall "package-name"

```suggestion
  "dependencies": {
    "cors": "^2.8.5",
    "dotenv": "^16.0.3",
    "express": "^4.18.2",
    "mysql2": "^2.3.3",
    "typeorm": "^0.3.10"
  },
  devDependencies: {
    "morgan": "^1.10.0",
    "nodemon": "^2.0.20"
  }
```